### PR TITLE
add docs and docstring for sonar_model in open_raw

### PR DESCRIPTION
**Pull request by leewujung**
This PR addresses #461 by improving documentation and docstring for `open_raw`.

### CODE REVIEW

File: docs/source/convert.rst [hunk 1]
```diff
@@ -37,11 +37,14 @@ File conversion for different types of echosounders is achieved by
...
 create a fully parsed ``EchoData`` object.
 
-For data files from EK60, EK80 and  EA640 echosounders,
-use the parameter ``sonar_model`` to indicate the echosounder type,
-since there is no specific information in the extension ``.raw``
-that includes information about the echosounder type.
-In this example, ``open_raw`` is used to convert a raw EK80 file,
+Use the parameter ``sonar_model`` to indicate the echosounder type:
+    - ``EK60``: Kongsberg Simrad EK60 echosounder
+    - ``EK80``: Kongsberg Simrad EK60 and Kongsberg EA640 echsoounders
```
Comment (emiliom) [inline docs/source/convert.rst:42]: I assume you meant **EK80**, not EK60

Comment (leewujung) [reply docs/source/convert.rst:42]: Oh right! thanks for catching it.

**Comment by emiliom** [comment]
It looks good now. I suggest you self merge once the checks complete. Thanks.

**Comment by gavinmacaulay** [comment]
Note that as of PR #409, data from Simrad ES70 and ES80 sounders are also supported via the sonar_model parameter (using 'ES70' and 'ES80').

**Comment by leewujung** [comment]
@gavinmacaulay : Yes! They will be added in v0.5.5 and we will add them to the docs then. This PR is to the `stable` branch which gets on the RTD page right away, as an independently branch that only updates the documentation. We will get v0.5.5 out very soon, later this week or early next week.

**Pull request merged**